pilot/pkg/features/xds.go

```diff
@@ -0,0 +1,113 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package features
+
+import (
+	"time"
+
+	"go.uber.org/atomic"
+
+	"istio.io/istio/pkg/env"
+)
+
+// Define flags that affect XDS config generation here.
```

Comment:
Not clear the distinction between this and experimental

Reply:
And these features should be in pilot

Reply:
I will move them to experimental for now and we can move to pilot once they are stable